<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KBPickStartDate</class>
 <widget class="QDialog" name="KBPickStartDate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pick Start Date</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTextBrowser" name="label">
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="autoFormatting">
      <set>QTextEdit::AutoAll</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="buttonGroup">
     <property name="title">
      <string>Start Date</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QRadioButton" name="noDateButton">
        <property name="text">
         <string>&amp;No date (let the bank determine the start date)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="lastUpdateButton">
        <property name="text">
         <string>&amp;Last Update</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="lastUpdateLabel">
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="firstDateButton">
        <property name="text">
         <string>F&amp;irst possible date</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="firstDateLabel">
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QRadioButton" name="pickDateButton">
        <property name="text">
         <string>Pick &amp;date</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="KMyMoneyDateEdit" name="pickDateEdit" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>KMyMoneyDateEdit</class>
   <extends>KDateComboBox</extends>
   <header>kmymoneydateedit.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>noDateButton</tabstop>
  <tabstop>lastUpdateButton</tabstop>
  <tabstop>firstDateButton</tabstop>
  <tabstop>pickDateButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pickDateButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>pickDateEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>308</y>
    </hint>
    <hint type="destinationlabel">
     <x>490</x>
     <y>308</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>KBPickStartDate</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>366</x>
     <y>380</y>
    </hint>
    <hint type="destinationlabel">
     <x>131</x>
     <y>402</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>KBPickStartDate</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>450</x>
     <y>379</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>401</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
